src/main/java/ru/hogwarts/school/repository/StudentRepository.java

```diff
@@ -0,0 +1,13 @@
+package ru.hogwarts.school.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import ru.hogwarts.school.entity.Student;
+
+import java.util.List;
+
+public interface StudentRepository extends JpaRepository<Student, Long> {
+    List<Student> findAllByAge(int age);
+    List<Student> findAllByBetween(int minAge, int maxAge);
```

Comment:
```suggestion
    List<Student> findAllByAgeBetween(int minAge, int maxAge);
```
метод работать не будет, не указано поле для фильтрации